### PR TITLE
Add mace-osaka24-large Model

### PR DESCRIPTION
**Pull request by TShiotaSS**
## Description

Dear Matbench Discovery Developers

This PR adds `MACE-Osaka24-large`, a single-head universal MLIP trained on the integrated dataset of organic (OFF23) and inorganic (MPtrj) systems.

Since our model is based on the `MACE` architecture, we utilized the scripts in models/mace for benchmarking. However, we encountered a format mismatch error when evaluating thermal conductivity using models/mace/test_mace_kappa.py and scripts/evals/kappa.py. To address this, we created models/mace/convert_json_to_2024_format_mace_osaka24_large.py to convert the data format and successfully completed the evaluation.

Best regards,

Tomoya Shiota

## Checklist

Please open PRs as `draft` and only mark as `ready to review` after checking off the following items:

- [x] I created a new folder and YAML metadata file `models/<arch_name>/<model_variant>.yml` for my submission. `arch_name` is the name of the architecture and `model_variant.yml` includes things like author details, training set names and important hyperparameters.
- [x] I added the new model as a new attribute [`Model.<model_variant>` enum](https://github.com/janosh/matbench-discovery/blob/57d0d0c8a14cd317/matbench_discovery/enums.py#L274) on the `Model` enum in `enums.py`.
- [x] I uploaded the energy/force/stress model prediction file for the WBM test set to Figshare or another cloud storage service (`<yyyy-mm-dd>-<model_variant>-preds.csv.gz`).
- [x] I uploaded the model-relaxed structures file to Figshare or another cloud storage service in [JSON lines format](https://jsonlines.org) (`<yyyy-mm-dd>-wbm-IS2RE-FIRE.jsonl.gz`). JSON Lines allows fast loading of small numbers of structures with `pandas.read_json(lines=True, nrows=100)` for inspection.
- [x] I uploaded the phonon predictions to Figshare or another cloud storage service (`<yyyy-mm-dd>-kappa-103-FIRE-<values-of-dist|fmax|symprec>.gz`).
- [ ] I have uploaded the diatomic predictions to Figshare or another cloud storage service (`<yyyy-mm-dd>-diatomics.json.gz`).
- [x] I included the urls to the Figshare files in the YAML metadata file (`models/<arch_name>/<model_variant>.yml`). If not using Figshare I have included the urls to the cloud storage service in the description of the PR.
- [x] I included the test script (`test_<arch_name>_<task>.py` for `task` in `discovery`, `kappa`, `diatomics`) that generated the prediction files.
- [ ] I have run `just prepare-model-submission <model_variant>` as described in the [contributing guide](https://github.com/janosh/matbench-discovery/blob/main/contributing.md) to check metadata and generate the plots needed for submission.
- [ ] I have installed/run the pre-commit hooks (`prek install` or `uvx prek`) and ensured all checks are passing.

Marking your PR as `ready to review` will trigger an automated code review, please address any issues raised by the automated review. For the maintainers minimizing the final human review process enables us to merge your submissions much faster!

- [ ] I included a training script (`train_<arch_name>.py`) if I trained a model specifically for this benchmark.
- [ ] I included a `readme.md` with additional details about my model.

### CODE REVIEW

**Comment by janosh** [comment]
thanks for the submission but closing this as stale for now. feel free to ping back!
